portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -46,14 +47,21 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import java.util.LinkedHashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
     int numberOfCommentsToDisplay = 0;
     
     @Override
-    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {      
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException { 
+      //Get number of comments.
+      numberOfCommentsToDisplay = getNumberOfCommentsToDisplay(request);
```

Comment:
May not be a problem but I know that when I got the number of comments to display in the doGet instead of the doPost, I ran into the problem where the variable would not update. Other than that LGTM